### PR TITLE
[FIX] fix nifti extension content access for  nibabel>5.3.0

### PR DESCRIPTION
**Pull request by bpinsard**
attempt @effigies suggested fix for #94

### CODE REVIEW

**Comment by bpinsard** [comment]
tests are not properly triggered on PR !?

**Comment by bpinsard** [comment]
solved in nibabel

**Comment by effigies** [comment]
@moloney I think it would be good to move to using public APIs even in subclasses, since private APIs are subject to change. But we'll keep this hack around for several versions at least. We'll start raising `DeprecationWarnings` before any eventual removal.

**Comment by Spenhouet** [comment]
> solved in nibabel

I have nibabel 5.3.2 installed but still get the `_content` error in dcmstack. Your comment and the note on the 5.3.1 release suggests this should not be the case: https://github.com/nipy/nibabel/releases/tag/5.3.1

File: src/dcmstack/dcmmeta.py [hunk 1]
```diff
@@ -570,7 +571,7 @@ def from_runtime_repr(klass, runtime_repr):
         dictionaries).
         '''
         result = klass(dcm_meta_ecode, '{}')
-        result._content = runtime_repr
+        result.set_content(runtime_repr)
```
Comment (Spenhouet) [inline src/dcmstack/dcmmeta.py:574]: At least in Nibabel 5.3.2 there is no `set_content`, but 
```
klass(dcm_meta_ecode, '{}', runtime_repr)
```
should work

**Pull request closed without merging**